projects/1.16/assets/materialis/materialis/lang/zh_cn.json

```
@@ -0,0 +1,259 @@
+  "material.materialis.pink_slime": "粉色黏液",
+  "material.materialis.pink_slime.encyclopedia": "每级降低10%的采掘速度和攻击速度，并获得300乘以消耗的数据作为黏液覆层的额外容量",
+  "material.materialis.plastic": "塑料",
+  "material.materialis.platinum": "铂",
+  "material.materialis.platinum.encyclopedia": "每级额外提升100%耐久消耗速度",
+  "material.materialis.psimetal": "Psi金属",
+  "material.materialis.psimetal.encyclopedia": "在工具采掘和攻击的时候施放选定的术式，并增加一个术式插槽",
+  "material.materialis.quicksilver": "水银",
+  "material.materialis.quicksilver.encyclopedia": "工具不再会因为使用而消耗耐久，但每秒有1%的概率消耗耐久",
+  "material.materialis.refined_glowstone": "强化荧石",
+  "material.materialis.refined_glowstone.encyclopedia": "采掘速度乘以属性等级除以20得出的值会添加到攻击速度中",
+  "material.materialis.refined_obsidian": "强化黑曜石",
+  "material.materialis.refined_obsidian.encyclopedia": "-20%接触距离",
+  "material.materialis.refined_radiance": "光辉石",
+  "material.materialis.refined_radiance.encyclopedia": "对目标造成20秒的发光效果，并在采掘方块后留下光源",
+  "material.materialis.refined_radiance.format": "光辉石 %s",
+  "material.materialis.rose_quartz": "玫瑰石英",
+  "material.materialis.rose_quartz.encyclopedia": "每个部件提供1个额外的升级槽",
+  "material.materialis.rose_quartz.flavor": "免责声明: 实际上并非使用玫瑰制成。",
+  "material.materialis.shadow_steel": "暗影钢",
+  "material.materialis.shadow_steel.encyclopedia": "击杀的生物和采掘的方块都不再掉落物品，但是击杀生物后会掉落额外的经验值",
+  "material.materialis.starmetal": "星辉",
+  "material.materialis.starmetal.encyclopedia": "工具会在午夜获得每级2点的采掘速度",
+  "material.materialis.starmetal.flavor": "不要与星形切割金属混淆。",
+  "material.materialis.tungsten": "钨",
+  "material.materialis.tungsten.encyclopedia": "工具的耐久度越低，耐久度的消耗几率越低，最低为 1%",
+  "material.materialis.uranium": "铀",
+  "material.materialis.uranium.encyclopedia": "耐久度在一半以上的时候会以双倍速度消耗耐久度，在低于一半耐久度的时候耐久度消耗速度则会越来越慢",
+  "material.materialis.utherium": "御腐",
+  "material.materialis.utherium.encyclopedia": "每级对腐衰生物造成25%的额外伤害",
+  "modifier.materialis.adrenaline": "肾上腺素",
+  "modifier.materialis.adrenaline.description": "生命值越低，攻击伤害越高",
+  "modifier.materialis.adrenaline.flavor": "或战或逃！",
+  "modifier.materialis.arcane": "奥术",
+  "modifier.materialis.arcane.description": "额外对目标造成少量魔法伤害",
+  "modifier.materialis.arcane.flavor": "魔法 *哼*",
+  "modifier.materialis.auxiliary_power": "备用能源",
+  "modifier.materialis.auxiliary_power.description": "采掘速度越高，攻击速度越高",
+  "modifier.materialis.auxiliary_power.flavor": "额外的果汁！",
+  "modifier.materialis.brittle": "易碎",
+  "modifier.materialis.brittle.description": "工具的耐久度会消耗得更快",
+  "modifier.materialis.brittle.flavor": "脆得跟粉笔一样",
+  "modifier.materialis.cleansing": "净化",
+  "modifier.materialis.cleansing.description": "对腐衰生物造成更多伤害",
+  "modifier.materialis.cleansing.flavor": "堕落的野兽！",
+  "modifier.materialis.colorized": "彩色",
+  "modifier.materialis.colorized.description": "为你的工具添加装饰性着色器",
+  "modifier.materialis.colorized.flavor": "五彩斑斓！",
+  "modifier.materialis.create_wrenching": "机械扳手", 
+  "modifier.materialis.create_wrenching.description": "工具可以通过潜行右键来拆卸机械动力的组件",
```

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

```suggestion
  "modifier.materialis.create_wrenching.description": "工具可以通过潜行右击来拆卸动力学组件",
```
右击，彻查文本中的“右键”“左键”问题。